src/librustdoc/clean/utils.rs

```diff
@@ -582,6 +583,8 @@ pub(crate) fn has_doc_flag(tcx: TyCtxt<'_>, did: DefId, flag: Symbol) -> bool {
 ///
 /// Set by `bootstrap::Builder::doc_rust_lang_org_channel` in order to keep tests passing on beta/stable.
 pub(crate) const DOC_RUST_LANG_ORG_CHANNEL: &str = env!("DOC_RUST_LANG_ORG_CHANNEL");
+pub(crate) static DOC_CHANNEL: Lazy<&'static str> =
```

Comment:
`DOC_RUST_LANG_ORG_CHANNEL` contains the full URL to the doc. It's not a problem in itself but since it's stored in all generated pages, better to just keep the channel, hence why I stored it this way.